### PR TITLE
Add play and speed to cli options

### PR DESCRIPTION
**Pull request by David-bfg**
A quality of life addition for the cli tool.
Add play argument to play TTS after it is generated.
Additionally include speed argument to be used with Coqui Studio models.

### CODE REVIEW

**Comment by erogol** [comment]
Thanks for the PR @David-bfg 

Would that work on different OSs? On linux you could just pipe to play commands I guess. It'd be better than introducing a new dependency.

**Comment by David-bfg** [comment]
> Would that work on different OSs?

https://simpleaudio.readthedocs.io/en/latest/
simpleaudio is cross platform win, linux & mac

> you could just pipe to play commands I guess.

Yes, I thought this would be preferable to just add a --pipe_out arg.

A lot or all of the logs are print lines so they would need to be suppressed to do so. I was not aware of such a feature beyond putting if statements around each log, but a cursory google search looks like there's something more manageable for that.

I'll look into it and circle back. Just hoping there is a similarly simple function to format the raw wav data to stdout as there is for saving it to a file.

File: TTS/bin/synthesize.py [hunk 1]
```diff
@@ -361,179 +362,177 @@ def main():
...
-    if args.play:
-        print(" > Playing TTS wav")
-
-    # save the results
-    print(" > Saving output to {}".format(args.out_path))
-    synthesizer.save_wav(wav, args.out_path, play=args.play)
+    pipe_out = sys.stdout if args.pipe_out else None
+    
+    with contextlib.redirect_stdout(None if args.pipe_out else sys.stdout):
+        # Late-import to make things load faster
```
Comment (David-bfg) [inline TTS/bin/synthesize.py:368]: just indentation changes from lines
368-417 and
420-531

**Comment by David-bfg** [comment]
@erogol any comments you could give on commit f1b1f4a

I went and looked into getting the wav file sent to standard out instead of the logs. This appears to be generally what I'd be looking to implement. Just wanted to check that the idea looked reasonable before cleaning it up and fully implementing.
Thanks.

**Comment by erogol** [comment]
@David-bfg I think piping with standard out makes sense. But we should drop the play argument to keep things simpler.

**Comment by David-bfg** [comment]
@erogol unless there are further code comments this should be complete or at it's last stage.

**Comment by erogol** [comment]
@David-bfg I'll review it next Monday. Thanks for the update.

**Comment by omega3** [comment]
I installed via 
`pip install TTS`
and 
`tts --text "companies seeking competitive advantage." --model_name "tts_models/en/vctk/vits" --speed 1.5 --out_path "$out" --speaker_idx p230`

shows
`tts: error: unrecognized arguments: --speed 1.5`

**Comment by David-bfg** [comment]
@omega3 https://pypi.org/project/TTS/ & latest readme show speed removed from docs. it only worked with ⓍTTS voice model if i recall.

**Pull request merged**